### PR TITLE
Create pokemon cell design

### LINKED ISSUE #6: Create Pokemon cell design
Create the base pokemon cell design for the list. The cell design should be the same as the figma.
Take in account only the design for only phones on its different sizes.

Image

### Acceptance Criteria

- [x] Create a class to handle the pokemon cell design
- [x] Create the required layout for the design
- [x] Make this cell and layout receive a pokemon object so we can use it when the list is built.
- [x] Validate with all the pokemon types
- [x] If #4 is done, associate the cell to the pokemon list.

### Epic Link
[Pokemon List Epic](https://github.com/jmacboy/Pokedex-ios/wiki/Epic-for-Pokemon-list)

### PR DESCRIPTION
**Pull request by Rodrigo-Schar**
pokemon cell design added, implemented pokemon model for set up data
closes #6

### CODE REVIEW

File: Pokedex_ios/Assets.xcassets/Background/Pokeball.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (jmacboy) [inline Pokedex_ios/Assets.xcassets/Background/Pokeball.imageset/Contents.json]: couldn't this be added as an svg?

Comment (jmacboy) [reply Pokedex_ios/Assets.xcassets/Background/Pokeball.imageset/Contents.json]: just checked below and is added double, couldn't the svg be used?

File: Pokedex_ios/Assets.xcassets/Pokemons/Bulbasaur.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (jmacboy) [inline Pokedex_ios/Assets.xcassets/Pokemons/Bulbasaur.imageset/Contents.json]: Is this image needed?

File: Pokedex_ios/View/Controller/PokemonListViewController.swift [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+        imageView.topAnchor.constraint(equalTo: margins.topAnchor, constant: -300).isActive = true
+        imageView.widthAnchor.constraint(equalTo: view.widthAnchor, constant: 1).isActive = true
+        imageView.heightAnchor.constraint(equalTo: imageView.widthAnchor, constant: 1).isActive = true
+        
+        self.view.sendSubviewToBack(imageView)
+    }
+    
+    func initViewModel() {
+        viewModel.getPokemons()
+        print(viewModel.pokemons)
```
Comment (jmacboy) [inline Pokedex_ios/View/Controller/PokemonListViewController.swift:47]: prints shouldn´t be committed

**Review by jmacboy — CHANGES_REQUESTED** [review]

File: Pokedex_ios/Assets.xcassets/Background/Pokeball.imageset/Contents.json (hunk 1 above)
Comment (Rodrigo-Schar) [reply Pokedex_ios/Assets.xcassets/Background/Pokeball.imageset/Contents.json]: the svg is being used i deleted the png

File: Pokedex_ios/Assets.xcassets/Pokemons/Bulbasaur.imageset/Contents.json (hunk 2 above)
Comment (Rodrigo-Schar) [reply Pokedex_ios/Assets.xcassets/Pokemons/Bulbasaur.imageset/Contents.json]: i deleted this image too you can check it again

**Review by jmacboy — APPROVED** [review]

**Pull request merged**